concordia/admin/actions.py

```diff
@@ -1,9 +1,25 @@
+import uuid
+
 from django.contrib import messages
 from django.utils.timezone import now
 
 from ..models import Asset, Transcription, TranscriptionStatus
 
 
+def anonymize_action(modeladmin, request, queryset):
+    count = queryset.count()
+    for user_account in queryset:
+        user_account.username = uuid.uuid4()
```

Comment:
Should we prefix this for easy filtering?

```suggestion
        user_account.username = "Anonymized %s" % uuid.uuid4()
```